test/unit/parser_compatability_test.rb

```diff
@@ -461,7 +466,7 @@ def find_operator(string)
     end
 
     # truckload of fail
-    ["BathsFull Bt 10,12.5", "DateField Bt 10,2012-12-31"].each do |f|
```

Comment:
Because of fix, thiis type of example is NOW SUPPORTED! I added an explicit example in the hash above.